Cargo.toml

```diff
@@ -144,3 +145,6 @@ name = "treeview"
 
 [[bin]]
 name = "list_store"
+
+[patch.crates-io]
+futures-preview = { git = "https://github.com/rust-lang-nursery/futures-rs.git"}
```

Comment:
This is needed because of the `future::Either` type we use in the non-async/await example. Will send a new PR once that one is merged.

Reply:
I mean once that one is uploaded to crates.io :)